pkg/controller/networkaddonsconfig/networkaddonsconfig_controller.go

```diff
@@ -277,7 +278,7 @@ func (r *ReconcileNetworkAddonsConfig) Reconcile(request reconcile.Request) (rec
 }
 
 // Render objects for all desired components
-func (r *ReconcileNetworkAddonsConfig) renderObjects(networkAddonsConfig *opv1alpha1.NetworkAddonsConfig, openshiftNetworkConfig *osv1.Network) ([]*unstructured.Unstructured, error) {
+func (r *ReconcileNetworkAddonsConfig) renderObjects(networkAddonsConfig *cnaov1.NetworkAddonsConfig, openshiftNetworkConfig *osv1.Network) ([]*unstructured.Unstructured, error) {
```

Comment:
If we pass `SchemeGroupVersion` and `cnao.NetworkAddonsConfigSpec` we don't need to refeerence cnaov1

Reply:
ditto